src/contract.js

```diff
@@ -103,12 +103,12 @@ class Contract {
           break;
         }
         case 'uint8': {
-          hex = Encoder.uint8ToHex(args[index]);
+          hex = Encoder.uintToHex(args[index]);
           dataHex = dataHex.concat(hex);
           break;
         }
         case 'uint256': {
-          hex = Encoder.uint256ToHex(args[index]);
+          hex = Encoder.padHexString(args[index]);
```

Comment:
@dwalintukan should we add a comment here because we r passing unit256 as a string to api so only padding here instead of conversion

Reply:
Yeah, so previously we have uint8ToHex() and uint256ToHex() to distinguish the two. Any particular reason we changed both names?

Reply:
@frankobe done.
@chrisli30 In the future, I need to make it more variable so if anyone passes in `uint16`, `uint24`, etc all the way up to `uint48` then it will accept it and use `uintToHex`. And anything from `uint56 - uint256` will use `padHexString`.